tofd/pars/R3BTofDCal2HitPar.h

```diff
@@ -193,6 +198,7 @@ class R3BTofDCal2HitPar : public FairTask
      */
     Double_t walk(Double_t Q, Double_t par1, Double_t par2, Double_t par3, Double_t par4, Double_t par5);
 
+
     /**
      * Method for calculation of saturation.
```

Comment:
I do not believe the saturation function is used. Are there plans to implement this or not? @lbott193

Reply:
I think TofdCal2Hit was introduced in PR#90 https://github.com/R3BRootGroup/R3BRoot/pull/90
There you can see that the saturation function is commented:
https://github.com/michael-heil/R3BRoot/blob/7f4b0d57dc2a9f2691dfdd7020a7da6a0553b967/tof/R3BTofdCal2Hit.cxx#L575-L577

I don't know how to track file changes on this webpage but I never used the saturation function, maybe @michael-heil can answer your question.